pinot-query-runtime/src/test/resources/queries/SpecialSyntax.json

```diff
@@ -0,0 +1,43 @@
+{
+  "special_syntax": {
+    "tables": {
+      "tbl1": [
+        {"name": "col1", "type": "STRING"},
+        {"name": "col2", "type": "STRING"},
+        {"name": "col3", "type": "INT"}
+      ],
+      "tbl2": [
+        {"name": "col1", "type": "STRING"},
+        {"name": "col2", "type": "STRING"},
+        {"name": "col3", "type": "INT"}
+      ]
+    },
+    "inputs": {
+      "tbl1": [
+        ["foo", "alice", 1],
+        ["bar", "bob", 2]
+      ],
+      "tbl2": [
+        ["foo", "bob", 3],
+        ["alice", "alice", 4]
+      ]
+    },
+    "queries": [
+      {
+        "description": "test join with using clause",
+        "sql": "SELECT {tbl1}.col1, {tbl2}.col3 FROM {tbl1} JOIN {tbl2} USING (col1)",
+        "outputs": [
+          ["foo", 3]
+        ]
+      },
+      {
+        "description": "test join with using clause but join with columns not being selected",
+        "sql": "SELECT {tbl1}.col1, {tbl2}.col3 FROM {tbl1} JOIN {tbl2} USING (col2)",
+        "outputs": [
```

Comment:
We may want to assert the output schema as well. Especially the output column names esp if queries are slightly complex (aliases / dot notation etc). Need not be done in this PR though.

Reply:
yeah this is a good question. we will add a alias/column reference test json separately to test these